plugins/module_utils/arguments.py

```diff
@@ -57,10 +57,14 @@
 )
 
 
-def get_mutation_payload(source, *wanted_params):
-    payload = {
+def get_spec_payload(source, *wanted_params):
```

Comment:
Can you add a few simple tests for this function? I know we are testing it indirectly already, but I would like to see some tests just for the sake of explaining what we expect this function to do.